docs/reference/storage-uri.md

```diff
@@ -30,12 +30,12 @@ The following are valid local file system URIs
 - ./quickwit
 ```
 
-<aside>
-⚠️ When using the `file://` protocol, a third `/` is necessary to express an absolute path.
+:::caution
```

Comment:
I think @fulmicoton add some dedicated css for <aside>, not sure if he wants to remove that. It's ok to keep the aside on a CSS point of view. Or did you see some issues?

Reply:
yes, the rendering is broken with <aside>

Reply:
Oh no I just did not know about :::caution. Does it look good?

Reply:
arf no :::caution is broken.